requirements.in

```diff
@@ -6,5 +6,6 @@ numpy!=1.15.0,!=1.16.0
 pandas>=0.23.0
 pyarrow>=0.11.1, <0.14.0 # Keep upper bound pinned until we see non-breaking releases in the future
 simplejson
+simplekv
 storefact
```

Comment:
Strictly speaking, I believe we do not require `storefact`, do we? As a test requirement, for sure, but not as a runtime requirement. Maybe we should switch this up. What do you think? An argument for keeping it would be that our (at least my) recommendation would be to use `storefact` but that's just a recommendation...

Reply:
It is also used in:

- benchmarks
- docstrings
- `kartothek.core.factory`

So I would like to keep that for now.